### PR TITLE
DERIVATIVO-24: Add support for image mirroring in rotation class

### PR DESCRIPTION
**Pull request by elohanlon**

### CODE REVIEW

**Comment by barmintor** [comment]
I was checking to see if this was about compliance level, but it's about reuse/spec completion, right?

File: lib/imogen/iiif/rotation.rb [hunk 1]
```diff
@@ -3,20 +3,24 @@ module Iiif
...
-        if rotation
-          yield img.rot("d#{rotation}")
-        else
-          yield img
-        end
+        angle, flip = Rotation.new(img).get(rotate)
+        # IIIF spec applies horizontal flip ("mirrored by reflection on the vertical axis") before rotation
+        img = img.fliphor if flip
+        # No need to rotate if angle is zero
+        img = img.rot("d#{angle}") unless angle.zero?
```
Comment (barmintor) [inline lib/imogen/iiif/rotation.rb:22]: Ah, thank you for this.

**Review by barmintor — APPROVED** [review]
Added a unit test for Rotation.convert, good changes here!

**Comment by elohanlon** [comment]
Reuse/spec completion was right, but thank you for the additional tests!  Even better!

**Pull request merged**